src/engine/controls/loopingcontrol.h

```diff
@@ -98,8 +86,7 @@ class LoopingControl : public EngineControl {
 
   signals:
     void loopReset();
-    void loopEnabledChanged(bool enabled);
-    void loopUpdated(mixxx::audio::FramePos startPosition, mixxx::audio::FramePos endPosition);
+    void loopUpdated(LoopInfo loopInfo);
```

Comment:
Why does clazy insist this should be fully qualified when used in engine controls?
It has no issue with `FrameInfo`.

Also, the reason `clazy-fully-qualified-moc-types` warns is
> not using fully-qualified type names [..] will break old-style connects and interaction with QML.

No old style here anymore, and is QML supposed to interact with engine controls??

Reply:
No idea. Does the proposed change work?

Reply:
Well, it builds (didn't run clazy locally), but I wonder if the reason is relevant here.
I'm not aware of any QML <--> EngineControl interaction, QML uses proxies for the library, players etc.
Control interaction is all done through _EngineBuffer_ and the individual controls interacting with _each other_.

So I'd rather ignore it and go with a clazy exclude for now
`// clazy:exclude=clazy-fully-qualified-moc-types`

Reply:
Though it's annoying that it has to be added everywhere it is used.